nxdrive/gui/folders_dialog.py

```diff
@@ -329,6 +348,43 @@ def _add_subgroup_duplicate_behavior(self, layout: QHBoxLayout, /) -> None:
         # Prevent previous objects to take the whole width, that does not render well for human eyes
         layout.addStretch(0)
 
+    def _new_folder_button_action(self) -> None:
+        """Update *new_folder_button* appearance and refresh *button_ok* state."""
+        css = self.CSS
+        label = "OK"
+        if self.new_folder.isReadOnly():
+            read_only_state = False
+            self.new_folder.setFocus(qt.MouseFocusReason)
+        else:
+            css = self.CSS_DISABLED
+            label = "Edit"
+            read_only_state = True
+
+        self.new_folder.setReadOnly(read_only_state)
+        self.new_folder_button.setText(Translator.get(label))
+        self.new_folder.setStyleSheet(css)
+        self.button_ok_state()
+
+    def _add_subgroup_new_folder(self, layout: QHBoxLayout, /) -> None:
+        """Add a sub-group for the new folder option."""
+        label = QLabel(Translator.get("NEW_REMOTE_FOLDER"))
+        label.setToolTip(Translator.get("NEW_REMOTE_FOLDER_TOOLTIP"))
+        label.setCursor(QCursor(Qt.WhatsThisCursor))
```

Comment:
```suggestion
        label.setCursor(QCursor(qt.WhatsThisCursor))
```
And add `WhatsThisCursor = Qt.CursorShape.WhatsThisCursor` to `qt/constants.py`.